creating_element/tutorial.md

```diff
@@ -41,24 +41,31 @@ def_input_pad :input,
   availability: :always,
   mode: :pull,
   demand_unit: :bytes,
-  caps: :any
+  caps: :any,
+  options: [
```

Comment:
I think we shouldn't mention pad options if there are no dynamic pads here, but I leave the decision up to you